definitions/contested/json/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -3660,6 +3660,34 @@
     "UserRole": "caseworker-divorce-financialremedy-courtadmin",
     "CRUD": "R"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "CaseFieldID": "bulkPrintLetterIdRes",
+    "UserRole": "caseworker-divorce-financialremedy-courtadmin",
+    "CRUD": "CRUD"
+  },
```

Comment:
only giving caseworker permission to see these consistent wit the fields in consented

Reply:
if any new fields/authorisations are identical with consent you can use the new common component to reduce duplication